elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -429,6 +431,9 @@ export class PeerPool extends EventEmitter {
 			ackTimeout: this._peerPoolConfig.ackTimeout,
 			wsMaxMessageRate: this._peerPoolConfig.wsMaxMessageRate,
 			wsMaxMessageRatePenalty: this._peerPoolConfig.wsMaxMessageRatePenalty,
+			maxPeerDiscoveryResponseLength: this._peerPoolConfig
```

Comment:
Does peer need to know this?

Reply:
The validation function which needs it is inside the peer list. It's a pure function so we can't pass it in another way.